types/actions/outputs/omrelp.pp

```diff
@@ -1,6 +1,7 @@
 # Struct data type for the Rsyslog RELP module's Action options
 type Rsyslog::Actions::Outputs::Omrelp = Struct[{
   target               => Variant[Stdlib::Fqdn, Stdlib::IP::Address],
+  port                 => Optional[Stdlib::Port],
```

Comment:
I'm not seeing a `port` option in the Rsyslog omrelp docs: https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html

Reply:
Yeah, the documentation is incomplete: https://github.com/rsyslog/rsyslog-doc/pull/831